doc/ref_model/chapters/chapter08-annex.md

```diff
@@ -139,3 +140,511 @@ Example, or Reference, Functest Status Reporting Artifacts:
+
+<a name="x.x.x"></a>
+### Resources
+1. **OPNFV** https://www.opnfv.org/ - project and community that facilitates a common NFVI, continuous integration (CI) with upstream projects, stand-alone testing toolsets, and a compliance and verification program for industry-wide testing and integration to accelerate the transformation of enterprise and service provider networks.<br>
+2. **CVC** https://wiki.lfnetworking.org/display/LN/Compliance+and+Verification+Committee - members-driven committee within LF Networking that recommends policies and oversight for compliance and certification program to the Governing Board of LF Networking (“Governing Board”).
+3. **Conducting OVP Testing with Dovetail** https://docs.opnfv.org/en/stable-danube/submodules/dovetail/docs/testing/user/userguide/testing_guide.html
+4. **Dovetail**
+ 1. Framework https://wiki.opnfv.org/display/dovetail/Dovetail+Test+Case+Requirements
+ 2. Test Plan: https://wiki.opnfv.org/display/dovetail/Dovetail+%28Danube%29+Documentation+for+Review?preview=/11698759/11698757/User%20Guide.pdf
+ 3. TCs:
+  1. https://wiki.opnfv.org/display/dovetail/Dovetail+%28Danube%29+Documentation+for+Review
+  2. Called by functest (repo): https://github.com/opnfv/dovetail/tree/master/etc/testcase
+  3. Per OVP release in the release notes:
+   1. https://docs.opnfv.org/en/stable-fraser/submodules/dovetail/docs/release/release-notes/index.html
+   2. https://docs.opnfv.org/en/stable-danube/submodules/dovetail/docs/release/release-notes/index.html
+5. **Overall documentation** is on docs.opnfv.org for the corresponding Fraser and Danube releases
+ 1. https://docs.opnfv.org/en/stable-fraser/testing/testing-user.html (Fraser)
+ 2. https://docs.opnfv.org/en/stable-fraser/testing/testing-dev.html (Fraser)
+6. **OPNFV Verification Program** is an open source, community-led compliance and verification program to demonstrate the readiness and availability of commercial NFV products and services, including NFVI and VNFs, using OPNFV and ONAP components (https://www.lfnetworking.org/OVP/).
+ 1. OVP Whitepaper - https://www.lfnetworking.org/resources/2019/04/03/ovp:-opnfv-verification-program/
+
+<a name="x.x"></a>
+## Lifecycle and Process Flow
+
+**NOT MVP**
+
+<a name="x.x.x"></a>
+### Project Mgmt
+
+**NOT MVP**
+
+How certifications are requested, processed, managed, what lab(s) is used, life cycle management
+
+<a name="x.x.x"></a>
+### OPNFV Iterations & Communications
+
+**NOT MVP**
+
+OPNNV Iterations with the CNTT (mgmt and communication of)
+
+<a name="x.x.x"></a>
+### Onboarding RA and Supplier VNF
+
+**NOT MVP**
+
+Onboarding (for OVP certification)
+
+<a name="x.x.x"></a>
+### SLAs and Issue Resolution
+
+**NOT MVP**
+
+Expectations for acknowlegment and turn-round from onboarding, verifications, and issue resolution.
+
+<a name="x.x.x"></a>
+### RA Verification
+
+**NOT MVP**
+
+Process flow for RA (Infra) Validation and Verfiication
+
+<a name="x.x"></a>
+## Current OVP and CVC Process
```

Comment:
Should be moved to RC 1 as it describes the LFN-based process.

Reply:
Yes, the entire General thoughts section has been commented out in Annex. we can move stuff  into RC1 as needed while RC1 is being developed (the text will not be lost as it is commented out). will resolve